docs/docsite/rst/configuring/configuring.rst

```diff
@@ -39,3 +39,17 @@ The following values are supported, and function identically to their CLI counte
       - run_this_tag
     use_default_rules: true
     verbosity: 1
+
+
+Pre-commit
```

Comment:
@awcrosby you can include chunks from readme instead of maintaining copy-pasted paragraphs :)